initializr-service/src/main/resources/application.yml

```diff
@@ -1256,6 +1263,17 @@ initializr:
           links:
            - rel: reference
              href: https://docs.spring.io/spring-shell/docs/2.0.0.M2/reference/htmlsingle/
+        - name: Spring Statemachine
+          id: spring-statemachine
+          groupId: org.springframework.statemachine
+          artifactId: spring-statemachine-starter
+          description: Statemachine integration
+          version: 2.0.0.M3
```

Comment:
You don't need a version if you have a bom

Reply:
Hmm, I explicitly removed starter from bom as I thought it doesn't belong there. I wonder if I should put it back.

Reply:
Yes you should. There is no reason for that one not to be there.

Reply:
ok, I'll create a new milestone and push all needed changes to this PR.